sdk/storage/azure-storage-file-share/tests/test_share.py

```diff
@@ -1818,6 +1818,49 @@ def test_set_share_properties_access_tier_premium(self, **kwargs):
         finally:
             self._delete_shares()
 
+    @FileSharePreparer()
+    @recorded_by_proxy
+    def test_provisioned_billing_v2(self, **kwargs):
```

Comment:
According to the OneNote, you need to mark these tests as Playback only because this is done on a special account type that isn't supported in ARM templates yet and therefore these tests will fail in the Live test pipeline.